test/unit/specs/components/Personalization/createApplyPropositions.spec.js

```diff
@@ -53,13 +53,13 @@ describe("Personalization::createApplyPropositions", () => {
     executeDecisions.and.returnValue(
       Promise.resolve(PAGE_WIDE_SCOPE_DECISIONS)
     );
```

Comment:
Please changes in this file. It just adds noise. If you want to fix it add a separate PR.

Reply:
@ninaceban It's just a minor fix, can we please leave it?